GliaWidgets/Public/Glia/Glia+StartEngagement.swift

```diff
@@ -22,17 +23,29 @@ extension Glia {
     ///
     public func startEngagement(
         engagementKind: EngagementKind,
+        in queueIds: [String],
         theme: Theme = Theme(),
         features: Features = .all,
         sceneProvider: SceneProvider? = nil
     ) throws {
+        // `interactor?.queueIds.isEmpty == false` statement is needed for integrators who uses old interface
+        // and pass queue identifier throught `configuration` function.
+        guard !queueIds.isEmpty || interactor?.queueIds.isEmpty == false else { throw GliaError.startEngagementWithNoQueueIds }
```

Comment:
Looks like this line is better to place after `guard let interactor = self.interactor else { throw GliaError.sdkIsNotConfigured }` line. 
What do you think?

Reply:
I prefer to validate passed values before doing anything with SDK and checking configuration.
Could you please clarify your point and tell why this validation is better after configuration check?

Reply:
My thoughts were not about priority of things to validate, but about interactor instance optionality. 
Your point sound reasonable, so let's omit my comment.